src/components/molecules/ProjectDetailsVerification/ProjectDetailsVerification.tsx

```diff
@@ -0,0 +1,40 @@
+import { SimpleGrid, Skeleton, Text, Button, HStack, Hide } from '@chakra-ui/react'
+import { useTranslation } from 'react-i18next'
+
+import { DetailWidget } from '@/components/atoms/DetailWidget/DetailWidget'
+import { formatDate } from '@/utils/DateFormat'
+import { DateFormats } from '@/@types/DateFormats'
+
+interface ProjectDetailsVerificationProps {
+  validation?: Validation
+}
+
+export const ProjectDetailsVerification = (props: ProjectDetailsVerificationProps) => {
+  const { validation } = props
+  const { t } = useTranslation(`projectDetails`)
+  if (!validation) {
+    return <Skeleton height="100px" />
+  }
+
+  return (
+    <SimpleGrid columns={2} gap="24px">
+      <DetailWidget title={t(`verificationHeaders.validationBody`)}>{validation.body}</DetailWidget>
+      <DetailWidget title={t(`verificationHeaders.validationUpdated`)}>{validation.date ? formatDate(validation.date, DateFormats.YYYY_MM_DD) : `--`}</DetailWidget>
```

Comment:
If `--` is the placeholder when there is no data, it should be part of i18n as well.